src/datomic_toolbox/core.clj

```diff
@@ -229,3 +229,24 @@
         {:created-at (first tx-instants)
          :updated-at (last tx-instants)
          :timestamps tx-instants})))
+
+(defn retry-tx
```

Comment:
It stands for "retry transaction". It cannot work with just taking
transaction data.

On Wed, Sep 9, 2015 at 1:59 PM, Christopher Shea <notifications@github.com>
wrote:

> In src/datomic_toolbox/core.clj
> <https://github.com/democracyworks/datomic-toolbox/pull/11#discussion_r39082258>
> :
>
> > @@ -229,3 +229,24 @@
> >          {:created-at (first tx-instants)
> >           :updated-at (last tx-instants)
> >           :timestamps tx-instants})))
> > +
> > +(defn retry-tx
>
> retry-tx sounds like it takes a transaction, like transaction data. But
> this is taking a function. Would this still work as just taking transaction
> data?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/democracyworks/datomic-toolbox/pull/11/files#r39082258>
> .
>